resources/measures/upgrade_hvac_add_heat_pump_rtu/measure.rb

```diff
@@ -75,6 +75,12 @@ def arguments(model)
     htg_to_clg_hp_ratio.setDescription('At rated conditions, a compressor will generally have slightly more cooling capacity than heating capacity. This factor integrates this ratio into the unit sizing.')
     args << htg_to_clg_hp_ratio
 
+    # model standard performance hp rtu
+    std_perf = OpenStudio::Measure::OSArgument.makeBoolArgument('std_perf', true)
+    std_perf.setDisplayName('Model standard performance HP RTU?')
```

Comment:
Can you add an argument describing what standard performance means? (2 stage cooling, single stage heating, etc.).